appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# https://github.com/sindresorhus/appveyor-node/blob/master/appveyor.yml
+
+environment:
+  matrix:
+    - platform: x64
+
+image: Visual Studio 2015
```

Comment:
@matheuss are you sure you need this, [VS Code's appveyor.yml](https://github.com/Microsoft/vscode/blob/master/appveyor.yml) doesn't have anything like it? Having said that, I don't know much about how images are used in appveyor.